es/utils/aepp-wallet-communication/wallet-connection/browser-runtime.js

```diff
@@ -29,26 +29,21 @@ import AsyncInit from '../../async-init'
 import { getBrowserAPI } from '../helpers'
 
 function connect (onMessage, onDisconnect) {
-  if (this.port) throw new Error('You already connected')
-  this.port = getBrowserAPI().runtime.connect(this.connectionInfo.id)
+  if (this.port.onMessage.hasListeners()) throw new Error('You already connected')
```

Comment:
`You are`
But I'd say not to say `you` but say what it is i.e. `client` or `wallet` and also add what it is connected to.